app/controllers/users/registrations_controller.rb

```diff
@@ -0,0 +1,71 @@
+# frozen_string_literal: true
+
+class Users::RegistrationsController < Devise::RegistrationsController
+  def new
+    @user = User.new
+  end
+
+  def create
+    @user = User.find_or_initialize_by(email: params.dig(:user, :email))
+    @external_user_profile = @user.external_user_profile || ExternalUserProfile.new
+
+    if email_taken?
+      send_magic_link(existing_user)
```

Comment:
Will this send a sign in email to an address that's already taken?

Reply:
I checked, yeahh all looks fine, just was half expecting an error of already in use rather than a link to sign up.

Reply:
yeah thats the intended behaviour